cves/kernel/CVE-2021-20265.yml

```diff
@@ -169,8 +176,11 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    I came to the conclusion of true because the fix commit noted that
+    the vulnerability was discovered by a type of fuzzer known as a
+    syzkaller program.
```

Comment:
Note could be shortened to something like 

```
Vulnerability was discovered using an automated fuzzing tool known as "Syzkaller"
```

Not mandatory, but I feel that more concise comments make these more readable